doc/3/essentials/network-loss-resilience/index.md

```diff
@@ -0,0 +1,19 @@
+---
+code: false
+type: page
+title: Network Loss Resilience
+description: Learn how to use the Kuzzle JAVA SDK with an instable network
+order: 400
+---
+
+# Network Loss Resilience
+
+The Kuzzle JAVA SDK provides tools that allow it to be used with an unstable network connection.
+
+## Automatic reconnection & resubscription
+
+The Kuzzle JAVA SDK can handle auto reconnection in case of a network disconnection and also resubscribe to all subscriptions if there is any.
```

Comment:
```suggestion
The Kuzzle JAVA SDK can automatically reconnect in case of a network disconnection and it can renew realtime subscriptions if there are any.
```